site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -0,0 +1,162 @@
+   ```
+1. Install the `kubectl` command-line tool by running the following command:
+   ```bash
+   gcloud components install kubectl
+   ```
+
+[gcloud-install]: https://cloud.google.com/sdk/docs/quickstarts
+
+## Creating the cluster
+
+A [cluster][cluster] consists of at least one *cluster master* machine and multiple worker machines called *nodes*: [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
+
+{{% feature expiryVersion="1.2.0" %}}
+```bash
+gcloud container clusters create [CLUSTER_NAME] --cluster-version=1.12 \
+  --tags=game-server \
+  --scopes=gke-default \
+  --num-nodes=4 \
+  --no-enable-autoupgrade \
+  --machine-type=n1-standard-4
+```
+{{% /feature %}}
+{{% feature publishVersion="1.2.0" %}}
+```bash
+gcloud container clusters create [CLUSTER_NAME] --cluster-version=1.13 \
+  --tags=game-server \
+  --scopes=gke-default \
+  --num-nodes=4 \
+  --no-enable-autoupgrade \
+  --machine-type=n1-standard-4
+```
+{{% /feature %}}
+
+Flag explanations:
+
+{{% feature expiryVersion="1.2.0" %}}
+* cluster-version: Agones requires Kubernetes version 1.12.
+{{% /feature %}}
+{{% feature publishVersion="1.2.0" %}}
+* cluster-version: Agones requires Kubernetes version 1.13.
+{{% /feature %}}
+* tags: Defines the tags that will be attached to new nodes in the cluster. This is to grant access through ports via the firewall created in the next step.
+* scopes: Defines the Oauth scopes required by the nodes.
+* num-nodes: The number of nodes to be created in each of the cluster's zones. Default: 4. Depending on the needs of your game, this parameter should be adjusted.
+* no-enable-autoupgrade: Disable automatic upgrades for nodes to reduce the likelihood of in-use games being disrupted.
+* machine-type: The type of machine to use for nodes. Default: n1-standard-4. Depending on the needs of your game, you may wish to [have smaller or larger machines](https://cloud.google.com/compute/docs/machine-types).
+
+_Optional_: Create a dedicated node pool for the Agones controllers. If you choose to skip this step, the Agones
+controllers will share the default node pool with your game servers which is fine for kicking the tires but is not
+recommended for a production deployment.
+
+```bash
+gcloud container node-pools create agones-system \
+  --cluster=[CLUSTER_NAME] \
+  --no-enable-autoupgrade \
+  --node-taints agones.dev/agones-system=true:NoExecute \
+  --node-labels agones.dev/agones-system=true \
+  --num-nodes=1
+```
+
+_Optional_: Create a node pool for [Metrics]({{< relref "../../Guides/metrics.md" >}}) if you want to monitor the
+ Agones system using Prometheus with Grafana or Stackdriver.
+
+```bash
+gcloud container node-pools create agones-metrics \
+  --cluster=[CLUSTER_NAME] \
+  --no-enable-autoupgrade \
+  --node-taints agones.dev/agones-metrics=true:NoExecute \
+  --node-labels agones.dev/agones-metrics=true \
+  --num-nodes=1
+```
+
+Flag explanations:
```

Comment:
It's a bit weird that there are flag explanations for the metrics node pool but not the agones-system node pool. Unless this is meant to cover both of them....

Reply:
I'm assuming it's meant to cover both of them, since they have the same flags, just with different values. Do you think we should list the same explanation twice?